magpy/gui/dialogclasses.py

```diff
@@ -2203,11 +2203,14 @@ def bindControls(self):
 
     def OnParameter(self, e):
         # open fit dlg
-        idx = int(self.absstreamComboBox.GetValue().split(':')[0])
+        streamidx = int(self.absstreamComboBox.GetValue().split(':')[0])
+        for idx in range(len(self.dictlst)):
+            if self.dictlst[idx]['streamidx'] == streamidx:
+                index = idx
```

Comment:
Is it possible for `index` to not be set, and how should that be handled?
Once found you could also `break` the for loop